tools/deployment/presto-clp/scripts/init.py

```
@@ -226,21 +229,91 @@ def _add_worker_env_vars(coordinator_common_env_file_path: Path, env_vars: Dict[
+        return False
+
+    return True
+
+
+def _configure_worker_memory_env_vars(env_vars: Dict[str, str]) -> bool:
+    """Adds memory-related environment variables for the worker."""
+
+    try:
+        total_memory_bytes = int(psutil.virtual_memory().total)
+    except (AttributeError, ValueError) as exc:
+        logger.error("Unable to determine total system memory: %s", exc)
```

Ruff (0.14.1)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)